openapi/openapi.yml

```
@@ -1076,7 +1076,7 @@ paths:
       - $ref: '#/components/parameters/postgres_database_reference'
     get:
       operationId: getPostgresCACertificatesByName
-      summary: Download CA certificates for a specific Postgres Database in a location with name
+      summary: Download CA certificates
```

The description here should mention PostgreSQL.

```suggestion
      summary: Download CA certificates for a PostgreSQL database
```